### PR TITLE
feat(consultation-portal): Kam 1329 fe change graphql to reflect on changes in millilagi

### PR DESCRIPTION
**Pull request by karlarnar**
# FE: Change graphql to reflect on changes in millilagi

https://veflausnir.atlassian.net/browse/KAM-1329
https://veflausnir.atlassian.net/browse/KAM-1326
https://veflausnir.atlassian.net/browse/KAM-1327

## What

- Change graphql queries for changes in api
- Translation strings for texts in Case.json
- Make a share-able component called BlowoutList for use for both stakeholders and relatedCases boxes and in turn removing the stakeholders component
- If advice is private, then it should not show the advice
- User can select if he wants the advice to be private, but only if the case allows advices to be private

## Why

Specify why you need to achieve this

## Screenshots / Gifs

image
image
image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/consultation-portal/screens/Case/Case.json [hunk 1]
```diff
@@ -171,5 +177,9 @@
       "text": "Umsagnarfrestur er liðinn.",
       "textCont": "Niðurstöður hafa verið birtar og samráði lokið."
     }
+  },
+  "adviceCard": {
+    "hiddenName": "",
```
Comment (jonagnar) [inline apps/consultation-portal/screens/Case/Case.json:182]: tómt, á kannski að vera þannig?

File: apps/consultation-portal/screens/Case/Case.tsx [hunk 2]
```diff
@@ -133,9 +134,12 @@ const CaseScreen = ({ chosenCase, caseId }: Props) => {
             order={[2, 2, 2, 3, 3]}
           >
             <Stack space={3}>
-              {!isMobile && <CaseStatusBox status={chosenCase.statusName} />}
+              <CaseStatusBox status={chosenCase.statusName} />
               {chosenCase?.stakeholders?.length > 0 && (
-                <Stakeholders chosenCase={chosenCase} />
+                <BlowoutList list={chosenCase.stakeholders} isStakeholder />
+              )}
+              {chosenCase?.relatedCases?.length > 0 && (
```
Comment (jonagnar) [inline apps/consultation-portal/screens/Case/Case.tsx:141]: Best practice advice: setja segðir í vel nefndar breytur og reference-a svo breytuna í JSX.
t.d.
const isStakeholdersNotEmpty = chosenCase?.stakeholders?.length > 0;
etc.

File: apps/consultation-portal/screens/Case/components/index.ts
Comment (jonagnar) [inline apps/consultation-portal/screens/Case/components/index.ts:1]: The curse of export default 🧙‍♂️

File: apps/consultation-portal/types/interfaces.ts
Comment (jonagnar) [inline apps/consultation-portal/types/interfaces.ts:1]: afhverju alltaf interface 😿

File: apps/consultation-portal/screens/Case/Case.json (hunk 1 above)
Comment (korsbacka) [reply apps/consultation-portal/screens/Case/Case.json:182]: nei, meikar meiri sense að taka það út og birta ekki neitt

File: apps/consultation-portal/screens/Case/Case.tsx (hunk 2 above)
Comment (korsbacka) [reply apps/consultation-portal/screens/Case/Case.tsx:141]: Miklu meira lesanlegri þannig, útfæri það þannig :)

**Pull request merged**